src/settings.rs

```diff
@@ -57,9 +81,14 @@ impl Default for Settings {
             actix_keep_alive: None,
             adm_endpoint_url: "".to_owned(),
             adm_country_ip_map: DEFAULT_ADM_COUNTRY_IP_MAP.to_owned(),
+            adm_max_tiles: 2,
+            adm_query_tile_count: 10,
             tiles_ttl: 15 * 60,
+            adm_settings: AdmSettings::default(),
```

Comment:
`config` won't parse this json for us, we have to do it ourselves (see how `adm_country_ip_map` is handled).

Reply:
Pity there is no easy way of testing this (parsing of env vars completely isolated from other tests)

Reply:
Hrm, I should also fix up the storage settings too then, thanks!